packages/strapi-design-system/src/Searchbar/Searchbar.js

```diff
@@ -43,7 +43,7 @@ const SearchbarWrapper = styled.div`
   }
 `;
 
-export const Searchbar = ({ name, children, value, onClear, clearLabel, ...props }) => {
+export const Searchbar = forwardRef(({ name, children, value, onClear, clearLabel, ...props }, ref) => {
```

Comment:
I am not sure you need the `forwardRef` as the `ref` is not forwarded.

Also, can you add a rule for eslint about the unused vars?